mongodbatlas/data_source_mongodbatlas_event_trigger_test.go

```diff
@@ -0,0 +1,78 @@
+package mongodbatlas
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/mwielbut/pointy"
+	"go.mongodb.org/realm/realm"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceMongoDBAtlasEventTrigger_basic(t *testing.T) {
+	SkipTest(t)
+	var (
+		resourceName = "mongodbatlas_event_trigger.test"
+		projectID    = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+		appID        = os.Getenv("MONGODB_REALM_APP_ID")
+		eventResp    = realm.EventTrigger{}
+	)
+	event := realm.EventTriggerRequest{
+		Name:       acctest.RandomWithPrefix("test-acc"),
+		Type:       "DATABASE",
+		FunctionID: os.Getenv("MONGODB_REALM_FUNCTION_ID"),
+		Disabled:   pointy.Bool(false),
+		Config: &realm.EventTriggerConfig{
+			OperationTypes: []string{"INSERT", "UPDATE"},
+			OperationType:  "LOGIN",
```

Comment:
OperationType Login is not needed for a DB trigger, please ensure the config includes what is required (unless you are trying to test if someone includes a parameter not needed as well as needed?)

Reply:
you're right, fixed it, thank you melissa